### DISCUSSION TITLE
Oauth Server vs Oauth Web Use

### DISCUSSION THREAD

**Discussion by davisj95** [benrwoodard/adobeanalyticsr#158] [Q&A] [answered]
As I follow through the setup process I noticed in Adobe that there is an option to create credentials for Oauth server-to-server instead of oauth web. In this configuration you can either generate an access token within adobe or it gives you the cURL command to generate one yourself. Is this something supported in this package? I'd love to use that instead of needing to reauthenticate every week or so for automations I'd like to build.

**Comment by benrwoodard** [comment, marked as answer]
This is a recent auth option that we have not created support for yet but is a priority.
